src/main/scala/org/globalforestwatch/summarystats/carbonflux_custom_area/CarbonCustomSummary.scala

```diff
@@ -0,0 +1,124 @@
+package org.globalforestwatch.summarystats.carbonflux_custom_area
+
+import cats.implicits._
+import geotrellis.contrib.polygonal.CellVisitor
+import geotrellis.raster._
+import org.globalforestwatch.summarystats.Summary
+import org.globalforestwatch.util.Geodesy
+
+import scala.annotation.tailrec
+
+/** LossData Summary by year */
+case class CarbonCustomSummary(
+                              stats: Map[CarbonCustomDataGroup, CarbonCustomData] = Map.empty
+                            ) extends Summary[CarbonCustomSummary] {
+
+  /** Combine two Maps and combine their LossData when a year is present in both */
+  def merge(other: CarbonCustomSummary): CarbonCustomSummary = {
+    // the years.combine method uses LossData.lossDataSemigroup instance to perform per value combine on the map
+    CarbonCustomSummary(stats.combine(other.stats))
+  }
+}
+
+object CarbonCustomSummary {
+  // CarbonFluxSummary form Raster[CarbonFluxTile] -- cell types may not be the same
+
+  implicit val mdhCellRegisterForCarbonFluxRaster1: CellVisitor[Raster[CarbonCustomTile], CarbonCustomSummary] =
+    new CellVisitor[Raster[CarbonCustomTile], CarbonCustomSummary] {
+
+      def register(
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)